VAMobile/src/utils/hooks/index.tsx

```diff
@@ -35,16 +35,32 @@ import { useTheme as styledComponentsUseTheme } from 'styled-components'
  */
 export const useError = (currentScreenID: ScreenIDTypes): boolean => {
   const { errorsByScreenID } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  return useDowntime(DowntimeScreenIDToFeature[currentScreenID]) || !!errorsByScreenID[currentScreenID]
+  const downtime = useDowntimeByScreenID(currentScreenID)
+  if (downtime) {
+    return true
+  }
+
+  return !!errorsByScreenID[currentScreenID]
 }
 
 export const useDowntime = (feature: DowntimeFeatureType): boolean => {
   const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
-  const mw = downtimeWindowsByFeature[feature]
-  if (!!mw && mw.startTime <= DateTime.now() && DateTime.now() <= mw.endTime) {
-    return true
-  }
-  return false
+  return featureInDowntime(feature, downtimeWindowsByFeature)
+}
+
+export const useDowntimeByScreenID = (currentScreenID: ScreenIDTypes): boolean => {
+  const { downtimeWindowsByFeature } = useSelector<RootState, ErrorsState>((state) => state.errors)
+  const features = ScreenIDToDowntimeFeatures[currentScreenID]
+  return oneOfFeaturesInDowntime(features, downtimeWindowsByFeature)
+}
+
+export const featureInDowntime = (feature: DowntimeFeatureType, downtimeWindows: DowntimeWindowsByFeatureType): boolean => {
+  const mw = downtimeWindows[feature]
+  return !!mw && mw.startTime <= DateTime.now() && DateTime.now() <= mw.endTime
+}
+
+export const oneOfFeaturesInDowntime = (features: DowntimeFeatureType[], downtimeWindows: DowntimeWindowsByFeatureType): boolean => {
+  return !!features?.some((feature) => featureInDowntime(feature as DowntimeFeatureType, downtimeWindows))
```

Comment:
I don't feel too strongly about this, but since this file is intended for hooks, and there are multiple helper functions for downtime, it might be worth making a separate file for downtime utils in the `VAMobile/src/utils` directory and moving these there

Reply:
I agree. I don't want to hold this up any further but have a followup ticket for this and will move these methods at that time.